docs/content/latest/yugabyte-platform/secure-universes/create-kms-config.md/aws-kms.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Create a KMS configuration using AWS KMS
+headerTitle: Create a KMS configuration using AWS KMS
+linkTitle: Create a KMS configuration
+description: Use Yugabyte Platform to create a KMS configuration for Amazon Web Services (AWS) KMS.
+aliases:
+  - /latest/yugabyte-platform/secure-universes/create-kms-config
+menu:
+  latest:
+    parent: secure-universes
+    identifier: create-kms-config-1-aws-kms
+    weight: 20
+isTocNested: true
+showAsideToc: true
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+
+  <li >
+    <a href="{{< relref "./aws-kms.md" >}}" class="nav-link active">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      AWS KMS
+    </a>
+  </li>
+
+  <li >
+    <a href="{{< relref "./equinix-smartkey.md" >}}" class="nav-link">
+      <i class="icon-cassandra" aria-hidden="true"></i>
+      Equinix SmartKey
+    </a>
+  </li>
+
+</ul>
+
+Encryption at rest uses universe keys to encrypt and decrypt universe data. You can use the Yugabyte Platform console to create key management service (KMS) configurations for generating the required universe keys for one or more YugabyteDB universes. Encryption at rest in Yugabyte Platform supports the use of [AWS Key Management Service (KMS)](https://aws.amazon.com/kms/), from Amazon Web Services (AWS).
```

Comment:
Encryption at rest also supports Equinix Smartkey. Should that be documented too?